community-edition-containers/README.md

```diff
@@ -13,24 +13,5 @@ If you find any issues, please post them on the customer support portal, [suppor
 
 ## Compatibility
 
-The compatibility between `pygluu-compose` and Gluu Server is listed below:
-
-| pygluu-compose    | Gluu Server   |
-|:-----------------:|:-------------:|
-| 1.19.x            | 4.5.5         |
```

Comment:
I suggest we keep this . Its actually very helpful. All we have to do here is link all < 4.5.5 to the old container repo.  We should have something similar for the pygluu